src/components/ListingsContainer.js

```diff
@@ -1,14 +1,42 @@
-import React from "react";
-// import ListingCard from "./ListingCard";
+import React, { useState, useEffect } from "react"
+import ListingCard from "./ListingCard"
+
+function ListingsContainer({
+  listings,
+  setListings,
+  searchedListings,
+  setSearchedListings,
+}) {
+  const deleteListing = (listing) => {
+    const options = {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+    }
+    fetch(`http://localhost:6001/listings/${listing.id}`, options).then(
+      (_resp) => {
```

Comment:
If you're not using the response, no need to define it at all. 
```suggestion
      () => {
```